src/akeneo/harness.yml

```diff
@@ -114,13 +118,16 @@ attributes:
         - "= (version_compare(@('akeneo.major_version'), '5', '>=')) ? '*/30 * * * * /cron-run-with-env.sh /app/bin/console pim:data-quality-insights:evaluations --env=prod' : '*/30 * * * * /cron-run-with-env.sh /app/bin/console pimee:data-quality-insights:evaluate-products --env=prod'"
         - 0 4 * * * /cron-run-with-env.sh /app/bin/console pim:versioning:purge --more-than-days 90 --no-interaction --force --env=prod
         - 0 2 * * * /cron-run-with-env.sh /app/bin/console pim:completeness:calculate --env=prod
+      jobs_post7:
+        - '*/10 * * * * /cron-run-with-env.sh /app/bin/console pim:job-automation:push-scheduled-jobs-to-queue --env=prod'
+      jobs: "= version_compare(@('akeneo.major_version'), '7', '>=') ? @('backend.cron.jobs_post7') : @('backend.cron.jobs_pre7')"
   frontend:
     path: /app
     build:
       when: '[ -f "package.json" ]'
       steps:
         - "= version_compare(@('akeneo.major_version'), '6', '>=') ? 'export NODE_OPTIONS=--openssl-legacy-provider' : ''"
-        - run yarn install --frozen-lockfile
+        - run yarn install
```

Comment:
wondering how I can avoid this.

It requires the lockfile to be fixed on first install